contrib/release/post-release.sh

```diff
@@ -65,6 +65,21 @@ main() {
 
     git checkout -b pr/$version-digests $version
     ${DIR}/pull-docker-manifests.sh "$@"
+
+    echo -e "$ersion\n" > $version-release-summary.txt
+    # Grab the release notes for the current release, stop before the next
+    # release. The start of line `## vX.Y.Z` lines will match in command.
+    tail -n+4 CHANGELOG.md | sed '/^## v.*$/q' >> $version-release-summary.txt
+    tail -n+2 digest-$version.txt >> $version-release-summary.txt
+    logecho "Creating Github draft release"
+    logrun hub release create -d -F $version-release-summary.txt $version
+    logecho "Browse to $RELEASES_URL to see the draft release"
+
+    if version_is_prerelease "$version"; then
+        # No digest updates for main branch for prereleases.
+        return
+    fi
```

Comment:
There's a subtlety here: This will *not* submit digest changes for `*-pre*` releases. This is because we expect these pre release versions to always be tagged on `main`, and we don't need to update the digests there.

However, it *will* submit digest changes PRs for `*-rc*` releases, assuming that those are made on branches. 

The biggest impact would be on the `vX.Y.0-rc.0` release, since it would attempt to prepare the digest changes for that release. If RC0 is prepared on main instead of the branch, then maybe the script will attempt to submit the digests PR to the v1.15 branch before it exists, and fail out...?

This matches the release workflows for main branch in https://github.com/cilium/release/pull/141 .

Reply:
Ah well actually the script below asks the release manager to confirm if that's what we want. So I think both cases are covered :+1: